.releaserc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I believe we will need to move this file into a `.config` folder. There is a breaking change in `semantic-release` `v23` https://github.com/semantic-release/semantic-release/releases/tag/v23.0.0.

Reply:
It looks like that is only the case for `.releaserc.config.js` files. `.json` should still work ok.